assembly/src/parsers/io_ops/mem_ops.rs

```diff
@@ -164,341 +138,385 @@ fn push_mem_addr(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), Assemb
 #[cfg(test)]
 mod tests {
     use super::{
-        super::{
-            parse_loadw, parse_pop, parse_popw, parse_push, parse_pushw, parse_storew,
-            tests::get_parsing_error, Felt,
-        },
+        super::{parse_mem_read, parse_mem_write, tests::get_parsing_error, Felt},
         AssemblyError, Operation, Token,
     };
 
     // TESTS FOR PUSHING VALUES ONTO THE STACK (PUSH)
     // ============================================================================================
```

Comment:
nit: this title needs to be update. In general, I'd probably just have 2 sections here:

1. Tests for reading from memory
2. Tests for writing into memory